### PR TITLE
Reviung34 shield definition

### PR DESCRIPTION
**Pull request by alexkang**
## Board/Shield Check-list

- [y] This board/shield is tested working on real hardware
- [y] Definitions follow the general style of other shields/boards upstream ([Reference](https://zmk.dev/docs/development/new-shield))
- [y] `.zmk.yml` metadata file added
- [y] Proper Copyright + License headers added to applicable files (Generally, we stick to "The ZMK Contributors" for copyrights to help avoid churn when files get edited)
- [y] General consistent formatting of DeviceTree files
- [y] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
- [y] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
- [n/a] If split, no name added for the right/peripheral half
- [y] Kconfig.defconfig file correctly wraps _all_ configuration in conditional on the shield symbol
- [y] `.conf` file has optional extra features commented out
- [y] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase (OSH/personal projects without general availability should create a zmk-config repo instead)

### CODE REVIEW

File: app/boards/shields/reviung34/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# REVIUNG34
+
+REVIUNG34 is a 33-34 key unibody split keyboard by [gtips](https://github.com/gtips). An in-stock version can be found at [Little Keyboards](https://www.littlekeyboards.com/products/reviung34-analyst-keyboard-kit).
+
+By default, the 2x1u layout is used. To use to the 1x2u layout, add the following to your keymap:
+```
+chosen {
+    zmk,matrix_transform = &single_2u_transform;
+};
+```
```
Comment (caksoylar) [inline app/boards/shields/reviung34/README.md:10]: You might want to add this to the default keymap, similar to https://github.com/zmkfirmware/zmk/blob/main/app/boards/arm/corneish_zen/corneish_zen.keymap#L12 (perhaps with an additional comment).

**Comment by alexkang** [comment]
@caksoylar gentle ping! Addressed adding the matrix transforms to the default keymap.

File: app/boards/shields/reviung34/boards/nice_nano_v2.overlay [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    spi3_sleep: spi3_sleep {
+        group1 {
+            psels = <NRF_PSEL(SPIM_MOSI, 0, 6)>;
+            low-power-enable;
+        };
+    };
+};
+
+&spi3 {
```
Comment (caksoylar) [inline app/boards/shields/reviung34/boards/nice_nano_v2.overlay:18]: Tiny nit, 4 space indents would be better in this node for consistency.

**Review by caksoylar — APPROVED** [review]
LGTM, noted one tiny nit.

File: app/boards/shields/reviung34/boards/nice_nano_v2.overlay (hunk 2 above)
Comment (alexkang) [reply app/boards/shields/reviung34/boards/nice_nano_v2.overlay:18]: Good catch, fixed!

File: app/boards/shields/reviung34/reviung34.keymap [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        // keymap accordingly!
+        // zmk,matrix_transform = &single_2u_transform;
+    };
+};
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        Base {
```
Comment (petejohanson) [inline app/boards/shields/reviung34/reviung34.keymap:27]: Another nitpick: although not required, our convention here is lower case IDs for the layer nodes.

**Review by petejohanson — CHANGES_REQUESTED** [review]
Just one minor tweak to the keymap nodes.

**Comment by alexkang** [comment]
Whoops sorry, lemme resolve some of my messy workspace issues.

**Comment by alexkang** [comment]
> Whoops sorry, lemme resolve some of my messy workspace issues.

Ok done! Should be good to review again!

**Review by petejohanson — APPROVED** [review]
Thanks!

**Comment by petejohanson** [comment]
Merged. In the future, if you've accidentally gotten some other feature PR included, it's preferred to do an interactive rebase and remove the offending commits, instead of appending a revert commit at the end.

Thanks again for the PR!

**Pull request merged**